@AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/weapon/fn_weaponGuncontrol.sqf

```diff
@@ -21,7 +21,11 @@ params ["_heli"];
 _curAcq = [_heli] call fza_fnc_targetingGetSightSelect;
 _curAcq = _curAcq + 1;
 if (_curAcq > 3) then {
-    _curAcq = 0;
+    if !(_heli animationPhase "fcr_enable" == 1) then {
```

Comment:
You should be able to sight select FCR without an FCR present - think about transmitted targets